fspiop/v1_0/openapi3/paths/bulkTransfers_ID_error.yaml

```diff
@@ -0,0 +1,49 @@
+put:
+  description: >-
+    If the server is unable to find or create a bulk transfer, or another
+    processing error occurs, the error callback `PUT /bulkTransfers/{ID}/error`
+    is used. The `{ID}` in the URI should contain the `bulkTransferId` that was
+    used for the creation of the bulk transfer (`POST /bulkTransfers`), or the
+    `{ID}` that was used in the `GET /bulkTransfers/{ID}` request.
+  summary: Return bulk transfer information error
+  tags:
+    - bulkTransfers
+  operationId: BulkTransfersErrorByID
```

Comment:
there is some inconsistency in naming operations, especially errors: `BulkTransfersErrorByID` vs `AuthorizationsByIDAndError`